packages/riipen-ui/src/components/Typography.jsx

```diff
@@ -127,7 +132,8 @@ class Typography extends React.Component {
     mobileBreakpoint: "sm",
     textAlign: "inherit",
     textTransform: "inherit",
-    variant: "body1"
+    variant: "body1",
+    wordBreak: "normal"
```

Comment:
Was undecided between this and breakword, or just no class added to Typography if wordBreak is not passed (i.e., no default prop)?